include/bluetooth/services/fast_pair/fast_pair.h

```diff
@@ -156,6 +156,17 @@ struct bt_fast_pair_info_cb {
  * functions. The Fast Pair operations require the enabled Bluetooth subsystem
  * (see the bt_is_ready function) and the load operation of the settings subsystem.
  *
+ * Mandatory callbacks must be registered before the user enables the Fast Pair module with
+ * this API. The mandatory callback set may depend on the chosen Fast Pair extension set
+ * that is selected in the application Kconfig configuration. The enable API fails with
+ * an error if mandatory callbacks are not registered. An example of a mandatory callback
+ * is the @ref bt_fast_pair_fmdn_ring_cb_register function defined in the API header of
+ * the Find My Device Network extension when this extension is configured to support
+ * ringing operations.
```

Comment:
Maybe we could explicitly inform that Fast by by itself does provide any mandatory callbacks?